examples/https_attestation/backend/src/main.rs

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Backend server for the HTTPS Attestation example.
+//! 
+//! Listens to plain HTTP connections, since TLS is terminated by the HTTPS Attestation Service.
+
+use anyhow::{anyhow, Context};
+use futures::future::FutureExt;
+use hyper::{
+    service::{make_service_fn, service_fn},
+    Body, Request, Response, Server,
+};
+use log::info;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Clone)]
+#[structopt(about = "HTTPS Attestation Backend")]
+pub struct Opt {
+    #[structopt(
+        long,
+        help = "Address to listen on for the HTTP server.",
+        default_value = "[::]:8081"
+    )]
+    http_listen_address: String,
+}
+
+// Example message expected from the Oak application.
```

Comment:
Not sure which Oak application, or why it is "expected". Isn't think just the message that this server returns to its client?